### PR TITLE
Add SDC examples execution in public CI

### PR DESCRIPTION
**Pull request by Vyacheslav-Smirnov**
- New option test-mode=exmples is added to test.py and used

### CODE REVIEW

File: buildscripts/test.py [hunk 1]
```diff
@@ -196,6 +199,56 @@
 
         format_print(f'Tests for {package_type} packages are PASSED')
 
+    # Execute examples
+    if test_mode == 'examples':
+        format_print(f'Run SDC examples')
```
Comment (densmirn) [inline buildscripts/test.py]: You don't need f-string here.

File: buildscripts/test.py [hunk 2]
```diff
@@ -196,6 +199,56 @@
...
+        sdc_examples = os.path.join(sdc_src, 'examples')
```
Comment (densmirn) [inline buildscripts/test.py]: I would advise to use `pathlib` module instead of `os` as a newer approach.

File: buildscripts/test.py (hunk 1 above)
Comment (Vyacheslav-Smirnov) [reply buildscripts/test.py]: Ok, will remove.

File: buildscripts/test.py (hunk 2 above)
Comment (Vyacheslav-Smirnov) [reply buildscripts/test.py]: I would like to leave os.path.join here due to changing this to pathlib require to make changes in all other places.
I think this should be done in separate PR.

Comment (densmirn) [reply buildscripts/test.py]: I'm not sure it will be done in separate PR. It was like a recommendation, so you we can skip it in this PR.

**Pull request merged**